src/lib/console.ts

```diff
@@ -1,6 +1,7 @@
 import type { EventData } from './MessageManager'
 
 export const onConsole = (e: EventData) => {
+  console.log(JSON.stringify(e));
```

Comment:
@NeOMakinG do we want this? see l.8 below

Reply:
😶‍🌫️